### PR TITLE
gc_woker: start safePointChecker before run worker.

### PR DESCRIPTION
**Pull request by disksing**
If tidb is not able to finish bootstrap in 100 seconds, the safePointChecker will not be started, then all transactions are not able to execute.

### CODE REVIEW

File: store/tikv/gc_worker.go [hunk 1]
```diff
@@ -130,15 +130,17 @@ func (w *GCWorker) StartSafePointChecker() {
 		for {
 			select {
 			case spCachedTime := <-time.After(d):
+				gcWorkerCounter.WithLabelValues("check_safepoint").Inc()
```
Comment (shenli) [inline store/tikv/gc_worker.go]: Should we distinguish success and fail metrics?

**Comment by ngaut** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**